ping-thing-client.mjs

```diff
@@ -79,32 +96,13 @@ async function pingThing() {
     try {
       try {
         const latestBlockhash = await getLatestBlockhash();
-        const transaction = pipe(
-          createTransactionMessage({ version: 0 }),
-          (tx) => setTransactionMessageFeePayer(signer.address, tx),
-          (tx) =>
-            setTransactionMessageLifetimeUsingBlockhash(
-              latestBlockhash,
-              tx
-            ),
-          (tx) =>
-            appendTransactionMessageInstructions(
-              [
-                getSetComputeUnitLimitInstruction({
-                  units: 500,
-                }),
-                getTransferSolInstruction({
-                  source: signer,
-                  destination: signer.address,
-                  amount: 5000,
-                }),
-              ],
-              tx
-            )
+        const transactionMessage = setTransactionMessageLifetimeUsingBlockhash(
+          latestBlockhash,
+          BASE_TRANSACTION_MESSAGE
```

Comment:
Here, you can combine the base message with the fresh blockhash to create the message you want to send.